web/auth/user_from_auth.ex

```diff
@@ -105,12 +105,7 @@ defmodule PhoenixGuardian.UserFromAuth do
   defp auth_and_validate(auth, repo) do
     case repo.get_by(Authorization, uid: uid_from_auth(auth), provider: to_string(auth.provider)) do
       nil -> {:error, :not_found}
-      authorization ->
-        if authorization.token == auth.credentials.token do
-          authorization
-        else
-          {:error, :token_mismatch}
-        end
+      authorization -> authorization
```

Comment:
What happens in this case though? We have an authorization that represents 'google' for user x. If the token is different it's still the same record correct? We would just need to update the token provided (I think)